packages/microeditor-envelope/src/__tests__/AppFormerGwtApi.test.ts

```diff
@@ -0,0 +1,33 @@
+import { AppFormerGwtApi } from '../gwt/AppFormerGwtApi';
+import { DummyEditor } from './DummyEditor';
+import * as AppFormer from "appformer-js-core";
+import { GwtEditor } from '../gwt/GwtEditor';
+
+class DummyGWTEditor {
+  private wrappedEditor: AppFormer.Editor;
+
+  constructor (wrappedEditor: AppFormer.Editor) {
+    this.wrappedEditor = wrappedEditor;
+  }
+  public get(): AppFormer.Editor {
+    return this.wrappedEditor;
+  }
+}
+
+const dummyEditor: DummyEditor = new DummyEditor();
+const dummyGWTEditor: DummyGWTEditor = new DummyGWTEditor(dummyEditor);
+const editorId:string = "dummy editor";
+const appFormerGwtApi: AppFormerGwtApi = new AppFormerGwtApi();
```

Comment:
I think you could remove the explicit types on these declarations, since they can all be inferred